.gitignore

```diff
@@ -1,3 +1,80 @@
+
```

Comment:
Is this basically [github's gitignore](https://github.com/github/gitignore/blob/master/Python.gitignore)?

Reply:
This one is from https://www.gitignore.io/api/pydev,python,eclipse